projects/fsastorefrontlib/src/cms-components/my-account/myaccount.module.ts

```diff
@@ -88,8 +88,8 @@ const routes: Routes = [
     path: null,
     canActivate: [AuthGuard, CmsPageGuard],
     data: {
-      cxRoute: 'addressBook',
-      pageLabel: 'address-book',
+      cxRoute: 'addressInfo',
+      pageLabel: 'address-info',
```

Comment:
If we change name of this route, shouldn't we change the name of the component itself?